### ISSUE TITLE
Image preview stops working in tmux

### ISSUE THREAD

**Issue by knutwalker** [sxyazi/yazi#142]
I'm testing on macOS with iTerm and I can see the image preview:

```
❯ echo $TERM, $TERM_PROGRAM, $XDG_SESSION_TYPE
xterm-256color, iTerm.app,
```

![image](https://github.com/sxyazi/yazi/assets/633183/f4305745-6d3d-4dd7-bd15-6e9e4e738128)

However, when attaching to a tmux session, I can no longer see an image preview:

![image](https://github.com/sxyazi/yazi/assets/633183/a1d607a4-7bc9-47e5-8262-4434552a0d6d)

In tmux the term is set as follows

```
❯ echo $TERM, $TERM_PROGRAM, $XDG_SESSION_TYPE
screen-256color, tmux,
```

however, even when setting those to the previous values, there is still no preview

```
❯ export TERM=xterm-256color
❯ export TERM_PROGRAM=iTerm.app
❯ echo $TERM, $TERM_PROGRAM, $XDG_SESSION_TYPE
xterm-256color, iTerm.app,
❯ yazi # still no preview
```

**Comment by sxyazi** [comment]
Tmux seems to require the escape code to be escaped again, and there is an issue that I'm not sure how to obtain the original  `TERM_PROGRAM` value

**Comment by sxyazi** [comment]
I added the `set-option` on my `tmux.conf`:

```bash
set-option -ga update-environment TERM_PROGRAM
```

and it seems not work to me:

```bash
echo $TERM_PROGRAM
tmux

tmux show-environment TERM_PROGRAM
-TERM_PROGRAM
```

**Comment by sxyazi** [comment]
I found it only doesn't work in my Kitty, which is strange.

I've tested it with iTerm2, Hyper, and WezTerm, and it works fine in those. This might be related to Kitty's shell integration; I'm not sure.

**Comment by sxyazi** [comment]
I think I have completed the adaptation. Please give it a try.

https://github.com/sxyazi/yazi/pull/147

**Comment by knutwalker** [comment]
Thanks, @sxyazi that was fast :)

I built from the PR and can confirm that image preview works for me

<img width="868" alt="image" src="https://github.com/sxyazi/yazi/assets/633183/6a114429-c9e5-48bb-847c-66ba56742d99">

**Comment by sxyazi** [comment]
Thanks for the testing.

BTW, Seems you are using a patched nerd font, not clear why iTerm2 won't properly render its rounded characters, you could disable it as per the [Wiki](https://github.com/sxyazi/yazi/wiki/I-don't-like-nerd%E2%80%90fonts!).

**Comment by farzadmf** [comment]
@sxyazi I'm using `tmux` with `WezTerm` on MacOS, and I updated my `tmux.conf` to have the lines mentioned in the [wiki](https://github.com/sxyazi/yazi/wiki/Image-preview-within-tmux), but image preview doesn't seem to be working inside of `tmux`.

Outside of `tmux`, I can see image previews; I was hoping you could help troubleshoot the issue

**Comment by sxyazi** [comment]
Hey @farzadmf, can you do a `tmux show-environment` and `echo $TERM, $TERM_PROGRAM, $XDG_SESSION_TYPE` within tmux, then paste their outputs here, let me know what's happening?

**Comment by sxyazi** [comment]
Oh forgot to mention, tmux support is a newly introduced feature, and it will be shipped with Yazi 0.1.5. You need to use the latest Git version to experience it in advance.

**Comment by farzadmf** [comment]
Sorry @sxyazi , I got busy and didn't get a chance to update. Makes sense about the version; will test with the new version and update here if it still doesn't work
